app/em_editor.py

```diff
@@ -487,6 +487,8 @@ def setTextBuffer(self, textBuffer):
       KEY_PAGE_DOWN: textBuffer.cursorNeutralPageDown,
       KEY_SHIFT_PAGE_UP: textBuffer.cursorSelectPageUp,
       KEY_SHIFT_PAGE_DOWN: textBuffer.cursorSelectPageDown,
+      KEY_ALT_SHIFT_PAGE_UP: textBuffer.cursorAltSelectPageUp,
+      KEY_ALT_SHIFT_PAGE_DOWN: textBuffer.cursorAltSelectPageDown,
 
       CTRL_A: textBuffer.selectionAll,
```

Comment:
Just chatting: context about removing "Alt" from the function name (in comment above), somethin' I'm going for is to have the key bindings be the keys in this dictionary and the value functions (actions) be named by what happens (with no mention of the key, if possible). 

This is so a different keybinding reads well. e.g. some other keybinding might use F9 to selectionAll, and it would be weird for the function name selectionAll to have Ctrl or A or F9 be part of the action function name.

An example is the CTRL_A: textBuffer.selectionAll, CTRL_A: doesn't say anything about what it does and textBuffer.selectionAll doesn't say anything about what key it's bound to (I think that's good).

Reply:
Oh okay, I see now. Thanks for the tip. I'll try to keep that in mind.